packages/flutter/lib/src/cupertino/text_field.dart

```diff
@@ -1213,7 +1222,9 @@ class _CupertinoTextFieldState extends State<CupertinoTextField> with Restoratio
           (true, true) => widget.suffix ?? _buildClearButton(),
           (false, true) => _buildClearButton(),
         };
-        return Row(children: <Widget>[
+        return Row(
+          crossAxisAlignment: widget.crossAxisAlignment,
```

Comment:
This will affect the vertical alignment of the text too, is that intended?

Reply:
Yap, if suffix or prefix constrains will be big.